velox/dwio/parquet/tests/duckdb_reader/ParquetReaderTest.cpp

```diff
@@ -96,6 +96,38 @@ TEST_F(ParquetReaderTest, readSampleFull) {
   assertReadExpected(*rowReader, expected);
 }
 
+TEST_F(ParquetReaderTest, readAsLowerCase) {
```

Comment:
This probably should be moved to native parquet reader test, this file is for DuckDB reader

Reply:
Because I also change duckdb_reader to support lowercase mode, so I add a DuckDB reader test here